datafusion/core/tests/sqllogictests/test_files/misc.slt

```diff
@@ -19,8 +19,8 @@
 
 # Show string normalization working
 # '' --> (empty)
-# '' --> NULL   (TODO: needs https://github.com/apache/arrow-rs/issues/3268)
+# '' --> NULL
 query II
 select 'foo', '', NULL
 ----
-foo (empty)
\ No newline at end of file
+foo (empty) NULL
```

Comment:
@xxchan You are so quickly! 

I noticed your new release and prepared to remove the issue about `NULL`. Then I found you done!

Thank you